src/ctia/task/migrate_es_stores.clj

```diff
@@ -0,0 +1,229 @@
+(ns ctia.task.migrate-es-stores
+  (:require [clj-momo.lib.es
+             [document :as es-doc]
+             [index :as es-index]]
+            [clojure.string :as string]
+            [clojure.tools.logging :as log]
+            [ctia
+             [init :refer [init-store-service! log-properties]]
+             [properties :as p]
+             [store :refer [stores]]]))
+
+(def default-batch-size 100)
+
+(def add-groups
+  "set a document group to [\"tenzin\"] if unset"
+  (map (fn [{:keys [groups]
+            :as doc}]
+         (if-not (seq groups)
+           (assoc doc :groups ["tenzin"])
+           doc))))
+
+(def fix-end-time
+  "fix end_time to 2535"
+  (map
+   (fn [{:keys [valid_time]
+        :as doc}]
+     (if (:end_time valid_time)
+       (update-in doc
+                  [:valid_time
+                   :end_time]
+                  #(string/replace %
+                                   #"2535"
+                                   "2525"))
+       doc))))
+
+(def available-operations
```

Comment:
I think we should be able to add more "jobs" or "operations" like this over time, perhaps in their own namespace, and when we successfully apply those jobs over a data store in a given environment (int/test/prod) we can make a record somewhere that we applied that job/operation on that environment at a given datetime.  This would allow us to very easily run the data migration "orchestrator/runner" (for lack of a better term off the top of my head) on a given environment, and it would retroactively apply *all* of the jobs/operations that had not yet been applied in that environment.

For example, I need to do a migration soon to deduplicate indicators.  It's important for when we start making mappings between indicators and Tool, Malware, Vulnerability, and Attack Pattern objects that we don't have lots of duplicate indicators.  This means finding all duplicates, designating one as the original, updating all relationship mappings that point at the duplicates so they point to the originals, and then remove the duplicates.  This is just *one* example of migration jobs we're going to need to add over time, and the jobs will need to be applied cumulatively across multiple environments.  It would be great to have a structure we could extend over time to define these jobs and to record their application in a given environment.